olympics/olympics.ipynb

```diff
@@ -0,0 +1,1511 @@
+{
```

Comment:
Wondering, do HoloViz examples usually clear the notebook first before pushing/merging? @maximlt

Reply:
Yes, I believe they should except in rare cases. It came up at the meeting today. Maxime is going to add a (pre-commit?) check to ensure the notebook outputs are cleared

Reply:
Unfortunately, that cannot be done in a `pre-commit` command as some notebooks on this repo must be committed with their outputs, those that can't be run on the CI (e.g. need to large datasets, compute not available, etc.). And I don't want to try to implement something that will clear these notebooks by mistake.

Instead, I'll add a new `validate` step to check that.